openstf/model/objective_creator.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+from typing import Union
+
+from openstf.enums import MLModelType
+from openstf.model.objective import XGBRegressorObjective
+
+
+class ObjectiveCreator:
+    OBJECTIVES = {
+        MLModelType.XGB: XGBRegressorObjective
+    }
+    @staticmethod
+    def create_objective(model_type: Union[MLModelType, str]):
```

Comment:
Nice to have type ints for the input, for the output you could define is as well

Reply:
thx, done!